src/www/app/outline_server_repository/access_key_serialization.ts

```diff
@@ -70,7 +70,7 @@ function parseShadowsocksSessionConfigJson(maybeJsonText: string): ShadowsocksSe
 export async function fetchShadowsocksSessionConfig(configLocation: URL): Promise<ShadowsocksSessionConfig> {
   let response;
   try {
-    response = await fetch(configLocation);
+    response = await fetch(configLocation, {cache: 'reload'});
```

Comment:
The `cache: 'reload'` only makes sense if we actually use the cache.
Since we are not using the cache, I think it's clearer if we use `no-store`.

`reload` would be better if we were falling back to the cache (with `only-if-cached`), but we are not doing that in this PR.